project.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I don't believe a top-level `project.json` is required. This file should probably be deleted. Also, `*.old.csproj` should probably go.